tools/c_coverage/c_coverage_report.py

```diff
@@ -143,39 +142,42 @@ def collect_stats(files, fd, pattern):
 
 
 if __name__ == '__main__':
-    parser = optparse.OptionParser(
-        usage="[options] callgrind_file(s)")
-    parser.add_option(
-        '-d', '--directory', dest='directory',
-        default='coverage',
-        help='Destination directory for output [default: coverage]')
-    parser.add_option(
-        '-p', '--pattern', dest='pattern',
-        default='numpy',
-        help='Regex pattern to match against source file paths [default: numpy]')
-    parser.add_option(
-        '-f', '--format', dest='format', default=[],
-        action='append', type='choice', choices=('text', 'html'),
-        help="Output format(s) to generate, may be 'text' or 'html' [default: both]")
-    (options, args) = parser.parse_args()
+    import argparse
+
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        'callgrind_file', nargs='+',
+        help='One or more callgrind files')
+    parser.add_argument(
+        '-d', '--directory', default='coverage',
+        help='Destination directory for output (default: %(default)s)')
+    parser.add_argument(
+        '-p', '--pattern', default='numpy',
+        help='Regex pattern to match against source file paths '
+        '(default: %(default)s)')
```

Comment:
Might indent this to align with the above, but YMMV and I do not care (similar below, but nvm...).
```suggestion
             '(default: %(default)s)')
```